LICENSE

```diff
@@ -1,6 +1,6 @@
-The MIT License (MIT)
+MIT License
 
-Copyright (c) 2014 Katrina Owen, _@kytrinyx.com
+Copyright (c) 2017 Exercism, Inc
```

Comment:
According to the wikipedia page, the approximation (c) appears to be acceptable.
https://en.wikipedia.org/wiki/Copyright_symbol#Digital_representation